.github/workflows/test.yml

```diff
@@ -0,0 +1,33 @@
+on: [push]
+
+jobs:
+  test-job:
+    runs-on: ubuntu-latest
+    name: Run script test
+    steps:
+      - uses: actions/checkout@v4.1.1
+
+      - name: Run script
+        run: >
+          ./script.sh
+        id: commit-lint
+
+      - name: Check 0 errors
+        run: >
+          test "${{ steps.commit-lint.outputs.errors }}" -eq "0"
+
+  test-action:
+    runs-on: ubuntu-latest
+    name: Run actions test
+    steps:
+      - uses: actions/checkout@v4.1.1
```

Comment:
Also here, `checkout` only fetches the last commit.

Reply:
Same as [previous reply](https://github.com/asperan/check-conventional-commits-action/pull/1#discussion_r1380202605)